x-pack/plugins/ml/kibana.jsonc

```diff
@@ -31,7 +31,8 @@
       "unifiedSearch",
       "savedObjectsFinder",
```

Comment:
it looks like `savedObjectsFinder` can be removed

Reply:
Changed it to a required bundle here [`30f7dab` (#159625)](https://github.com/elastic/kibana/pull/159625/commits/30f7dabe60780faf434dd3cb0746e773507ee0c7)